backend/app/services/implementations/log_records_service.py

```diff
@@ -152,15 +152,16 @@ def get_log_records(
             logs.attn_to,\n \
             logs.note,\n \
             logs.tags,\n \
-            logs.building,\n \
+            buildings.id AS building_id,\n \
```

Comment:
so instead of selecting the building ID we'd want to select the name:
```
buildings.name AS building_name
```

for example

Reply:
resolved to be addressed later